### PR TITLE
Document AddDays exception

### PR DESCRIPTION
**Pull request by ploeh**
As requested by @tarekgh: https://github.com/dotnet/runtime/pull/94635#issuecomment-1806914210

### CODE REVIEW

File: xml/System/DateOnly.xml [hunk 1]
```diff
@@ -1,2442 +1,2445 @@
...
+        <ReturnType>System.DateOnly</ReturnType>
+      </ReturnValue>
+      <Parameters>
+        <Parameter Name="value" Type="System.Int32" />
+      </Parameters>
+      <Docs>
+        <param name="value">The number of days to add. To subtract days, specify a negative number.</param>
+        <summary>Adds the specified number of days to the value of this instance.</summary>
+        <returns>An instance whose value is the sum of the date represented by this instance and the number of days represented by value.</returns>
+        <exception cref="ArgumentOutOfRangeException">
```
Comment (tarekgh) [inline xml/System/DateOnly.xml:153]: Please do it as it is done in https://github.com/dotnet/dotnet-api-docs/blob/173aa811f0080f5573c370e43852c1c61bed18d0/xml/System/DateTime.xml#L977. 

Also, please revert any white spaces changes in the file you are editing. Thanks!

Comment (ploeh) [reply xml/System/DateOnly.xml:153]: Sorry, I was just using the GitHub web editor for this, so it must have been making all the whitespace changes...

Comment (tarekgh) [reply xml/System/DateOnly.xml:153]: No worries. I'll wait for your update and then we can proceed. Thanks!

Comment (ploeh) [reply xml/System/DateOnly.xml:153]: I'm going to send another pull request in order to avoid Git-history trashing.

**Pull request closed without merging**